test/lit.cfg

```diff
@@ -863,8 +867,9 @@ elif run_os in ['windows-msvc']:
             ('%r -modulewrap -target %s' % (config.swiftc, config.variant_triple))
 
 
-elif run_os in ['linux-gnu', 'linux-gnueabihf', 'freebsd', 'windows-cygnus', 'windows-gnu']:
-    # Linux/FreeBSD/Cygwin
+elif run_os in ['linux-gnu', 'linux-gnueabihf', 'freebsd', 'windows-cygnus', 'windows-gnu'] or \
+     (run_android and run_os in ['linux-android', 'linux-androideabi']):
```

Comment:
Alright, hack gone and put in a check so that lit can differentiate between running on Android or sending the tests to Android using adb.